examples/with-vitest/vitest.config.ts

```diff
@@ -10,7 +10,7 @@ export default defineConfig({
     },
     environment: "jsdom",
     globals: true,
-    setupFiles: ['node_modules/@testing-library/jest-dom/extend-expect', './setupVitest.js'],
+    setupFiles: ['./setupVitest.ts'],
     transformMode: { web: [/\.[jt]sx?$/] },
```

Comment:
According to my eslint the new vitest version should be `testTransformMode: { web: ['/.[jt]sx?$/'] },`. Is my assumption correct?

Reply:
I've made it a string array instead now.